gobblin-metrics-libs/gobblin-metrics-base/src/main/avro/GaaSObservabilityEventExperimental.avsc

```diff
@@ -208,12 +208,17 @@
               {
                 "name": "bytesWritten",
                 "type": "long",
-                "doc": "Number of bytes written for the dataset"
+                "doc": "Number of bytes written for the dataset, can be -1 if unsupported by the writer (e.g. jdbc writer)"
               },
               {
-                "name": "recordsWritten",
+                "name": "entitiesWritten",
                 "type": "long",
-                "doc": "Number of records written for the dataset"
+                "doc": "Number of entities written for the dataset by the Gobblin writer"
+              },
+              {
+                "name": "datasetCommitSucceeded",
```

Comment:
nit: do we really want to repeat 'dataset', as in `datasetsWritten[*].datasetCommitSucceeded` (e.g. we don't name it `datasetBytesWritten`)?  maybe `wasCommitted`(?)